packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -146,6 +146,67 @@ const proxy = resource.addProxy({
 });
 ```
 
+### Models
+
+You may define models to provide structure to request payloads sent to a method,
+and/or payloads returned to a client from a method. A model may be added to a
+REST API as follows:
+
+```ts
+// Direct addition to REST API
+const beerModel = api.addModel('Beer', 'application/json', {
```

Comment:
Use keyword arguments instead of positional (basically extract the common options from ModelProps and reuse here)

Reply:
Done.